### PR TITLE
Correct some looks on FF

### PR DESCRIPTION
**Pull request by jasonfellows**
* Removing the native arrow on FF select
* Remove our up/down arrows on FF since we can't remove their native spinner for number inputs
* Correct height of native select
* Remove option outline on focus in FF native select

### CODE REVIEW

File: src/scss/elements/_forms.scss [hunk 1]
```diff
@@ -1,6 +1,7 @@
 input,
 select,
 textarea {
+  -moz-appearance: none;
```
Comment (keithpops) [inline src/scss/elements/_forms.scss]: Just an open question but shouldn't our prefixer be taking care of this?

Comment (jasonfellows) [inline src/scss/elements/_forms.scss]: Should be, but it wasn't? I didn't did into those docs, but this addresses the issue.

Comment (keithpops) [inline src/scss/elements/_forms.scss]: I think I vaguely remember discovering `appearance` may not have been supported just yet from a while ago. I'll see if bumping a dependency somewhere might fix this in the future.

**Comment by keithpops** [comment]
+1

**Comment by loudhearted** [comment]
:+1:

**Pull request merged**